packages/cohere/src/cohere-chat-language-model.ts

```diff
@@ -192,6 +207,11 @@ export class CohereChatLanguageModel implements LanguageModelV1 {
       completionTokens: Number.NaN,
     };
 
+    let toolCallIndex = -1;
+    let toolCallId = '';
```

Comment:
do you need `toolCallId` when there are `toolCallIds`? Or would it be easier / less error prone to always use `toolCallIds`? Also, does `toolName` also need to be changed into an array? (unclear why we need `toolCallId` / `toolName` - the array makes sense to me bc there can be multiple tool calls)

Reply:
I agree that using an array would be best here – updated!